### PR TITLE
Log dagbag metrics dupplicate messages in queue into Statsd

### PR DESCRIPTION
**Pull request by mistercrunch**
@aoen @plypaul

### CODE REVIEW

File: airflow/models.py [hunk 1]
```diff
@@ -1122,6 +1129,7 @@ def run(
                 "Task {self} previously succeeded"
                 " on {self.end_date}".format(**locals())
             )
+            Stats.incr('previously_succeeded', 1, 1)
```
Comment (aoen) [inline airflow/models.py:1132]: maybe attempts_to_rerun_succeeded_tasks or something a bit more clear

File: airflow/models.py [hunk 2]
```diff
@@ -403,6 +406,10 @@ def collect_dags(
                                 filepath, only_if_updated=only_if_updated)
                     except Exception as e:
                         logging.warning(e)
+        Stats.gauge(
+            'collect_dags', (datetime.now() - start_dttm).total_seconds(), 1)
```
Comment (aoen) [inline airflow/models.py:410]: maybe collect_dags_seconds so that it's clear it's a time measurement

**Comment by aoen** [comment]
LGTM

File: airflow/models.py (hunk 2 above)
Comment (plypaul) [inline airflow/models.py:410]: dag_collection_time_seconds? Also, I'm not too familiar with how these metrics appear on the user end. Is it just this string or is there any additional context about which module it came from?

**Comment by plypaul** [comment]
Typo in title, but otherwise looks good.

File: airflow/models.py [hunk 3]
```diff
@@ -403,6 +406,10 @@ def collect_dags(
...
+        Stats.gauge(
+            'dagbag_size', len(self.dags), 1)
```
Comment (plypaul) [inline airflow/models.py:412]: Could you also add `Stats.gauge('import_errors', len(self.import_errors), 1)`?

**Pull request merged**